Gruntfile.js

```diff
@@ -169,29 +95,6 @@ module.exports = function(grunt) {
 
     grunt.registerTask('build', [
         'requirejs',
-        'filerev',
```

Comment:
This might not be the right PR for this, but I understand that grunt-filerev is deprecated. Unfortunately we need something do to revving to act as a cache-buster for new versions. I'm not sure what the best way is. Maybe factor that into an npm script around the requirejs build script?

Reply:
There wasn't a config or task specified that corresponded to the `filerev` name, so the filerev module wasn't being used. Checking the history of the file, the `filerev` config segment got removed four years ago (commit b2e62c573b97d9a69f4ba055bc9829463477b4cc) but the other bits of the gruntfile didn't get updated.

Reply:
Huh. Okay then. Works for me.